File: ncDados/dLoja.vb
Namespace nsLoja

  Public Class ColecaoLoja
    Inherits List(Of dLoja)
  End Class

  Public Class dLoja

    Private _cid As Nullable(Of Integer)
    Private _codigo As String
    Private _nomeFantasia As String
    Private _logradouro As String
    Private _numero As Nullable(Of Integer)
    Private _complemento As String
    Private _bairro As String
    Private _cidade As String
    Private _estado_cid As Nullable(Of Integer)
    Private _cep As Nullable(Of Integer)
    Private _cnpj As String
    Private _ddd As Nullable(Of Integer)
    Private _telefone As Nullable(Of Integer)
    Private _ramal As Nullable(Of Integer)
    Private _nomeContato As String
    Private _razaoSocial As String
    Private _situacao As String
    Private _spc_codigo_associado As String
    Private _spc_nome_informante As String
        Private _spc_controle_informante As String
        Private _inscestadual As String



        Public Property cid() As Nullable(Of Integer)
      Get
        Return _cid
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _cid = value
      End Set
    End Property

    Public Property nomeFantasia() As String
      Get
        Return _nomeFantasia
      End Get
      Set(ByVal value As String)
        _nomeFantasia = value
      End Set
    End Property

    Public Property codigo() As String
      Get
        Return _codigo
      End Get
      Set(ByVal value As String)
        _codigo = value
      End Set
    End Property

    Public Property logradouro() As String
      Get
        Return _logradouro
      End Get
      Set(ByVal value As String)
        _logradouro = value
      End Set
    End Property

    Public Property numero() As Nullable(Of Integer)
      Get
        Return _numero
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _numero = value
      End Set
    End Property

    Public Property complemento() As String
      Get
        Return _complemento
      End Get
      Set(ByVal value As String)
        _complemento = value
      End Set
    End Property

    Public Property bairro() As String
      Get
        Return _bairro
      End Get
      Set(ByVal value As String)
        _bairro = value
      End Set
    End Property

    Public Property cidade() As String
      Get
        Return _cidade
      End Get
      Set(ByVal value As String)
        _cidade = value
      End Set
    End Property

    Public Property estado_cid() As Nullable(Of Integer)
      Get
        Return _estado_cid
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _estado_cid = value
      End Set
    End Property

    Public Property cep() As Nullable(Of Integer)
      Get
        Return _cep
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _cep = value
      End Set
    End Property

    Public Property cnpj() As String
      Get
        Return _cnpj
      End Get
      Set(ByVal value As String)
        _cnpj = value
      End Set
    End Property

    Public Property ddd() As Nullable(Of Integer)
      Get
        Return _ddd
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _ddd = value
      End Set
    End Property

    Public Property telefone() As Nullable(Of Integer)
      Get
        Return _telefone
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _telefone = value
      End Set
    End Property

    Public Property ramal() As Nullable(Of Integer)
      Get
        Return _ramal
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _ramal = value
      End Set
    End Property

    Public Property nomeContato() As String
      Get
        Return _nomeContato
      End Get
      Set(ByVal value As String)
        _nomeContato = value
      End Set
    End Property

    Public Property razaoSocial() As String
      Get
        Return _razaoSocial
      End Get
      Set(ByVal value As String)
        _razaoSocial = value
      End Set
    End Property

    Public Property situacao() As String
      Get
        Return _situacao
      End Get
      Set(ByVal value As String)
        _situacao = value
      End Set
    End Property

    Public Property spc_codigo_associado() As String
      Get
        Return _spc_codigo_associado
      End Get
      Set(ByVal value As String)
        _spc_codigo_associado = value
      End Set
    End Property

    Public Property spc_nome_informante() As String
      Get
        Return _spc_nome_informante
      End Get
      Set(ByVal value As String)
        _spc_nome_informante = value
      End Set
    End Property

        Public Property spc_controle_informante() As String
            Get
                Return _spc_controle_informante
            End Get
            Set(ByVal value As String)
                _spc_controle_informante = value
            End Set
        End Property
        Public Property Inscestadual() As String
            Get
                Return _inscestadual
            End Get
            Set(ByVal value As String)
                _inscestadual = value
            End Set
        End Property






    End Class

End Namespace

File: ncModelos/dLoja.cs
using System;
using System.Collections.Generic;

namespace nsLoja
{
    public class ColecaoLoja : List<dLoja>
    {
    }

    public class dLoja
    {
        private int? _cid;
        private string _codigo;
        private string _nomeFantasia;
        private string _logradouro;
        private int? _numero;
        private string _complemento;
        private string _bairro;
        private string _cidade;
        private int? _estado_cid;
        private int? _cep;
        private string _cnpj;
        private int? _ddd;
        private int? _telefone;
        private int? _ramal;
        private string _nomeContato;
        private string _razaoSocial;
        private string _situacao;
        private string _spc_codigo_associado;
        private string _spc_nome_informante;
        private string _spc_controle_informante;
        private string _inscestadual;

        public int? cid
        {
            get => _cid;
            set => _cid = value;
        }

        public string nomeFantasia
        {
            get => _nomeFantasia;
            set => _nomeFantasia = value;
        }

        public string codigo
        {
            get => _codigo;
            set => _codigo = value;
        }

        public string logradouro
        {
            get => _logradouro;
            set => _logradouro = value;
        }

        public int? numero
        {
            get => _numero;
            set => _numero = value;
        }

        public string complemento
        {
            get => _complemento;
            set => _complemento = value;
        }

        public string bairro
        {
            get => _bairro;
            set => _bairro = value;
        }

        public string cidade
        {
            get => _cidade;
            set => _cidade = value;
        }

        public int? estado_cid
        {
            get => _estado_cid;
            set => _estado_cid = value;
        }

        public int? cep
        {
            get => _cep;
            set => _cep = value;
        }

        public string cnpj
        {
            get => _cnpj;
            set => _cnpj = value;
        }

        public int? ddd
        {
            get => _ddd;
            set => _ddd = value;
        }

        public int? telefone
        {
            get => _telefone;
            set => _telefone = value;
        }

        public int? ramal
        {
            get => _ramal;
            set => _ramal = value;
        }

        public string nomeContato
        {
            get => _nomeContato;
            set => _nomeContato = value;
        }

        public string razaoSocial
        {
            get => _razaoSocial;
            set => _razaoSocial = value;
        }

        public string situacao
        {
            get => _situacao;
            set => _situacao = value;
        }

        public string spc_codigo_associado
        {
            get => _spc_codigo_associado;
            set => _spc_codigo_associado = value;
        }

        public string spc_nome_informante
        {
            get => _spc_nome_informante;
            set => _spc_nome_informante = value;
        }

        public string spc_controle_informante
        {
            get => _spc_controle_informante;
            set => _spc_controle_informante = value;
        }

        public string Inscestadual
        {
            get => _inscestadual;
            set => _inscestadual = value;
        }
    }
}
